IanGRap/Squish
# Extensive Scene Updates

Goal : Visually overhaul the scene to give it visual interest to the player.

Who Worked On It: Reed Scriven

What Changed: A lot, to avoid getting too specific here's the things that change that are the most connected to the rest of the game 
- Rotated block meshes inside of the main layout as well as in the block pattern prefabs
- Added a background to the character select
- added in new models from the artists
- changed materials and scene lighting settings
- made a LOT of scene changes. Hopefully this won't get messy

How to Test:
- Test block pushing and make sure the alignment bug doesn't happen again. I did some testing but we should be absolutely sure. 
- Does the merge break with all the scene changes?
- Is the background too distracting if so, set fog to .05 to 0.06 

Notes: Is there a better way to do "what changed" for art stuff?
